utils/superset/README.md

```diff
@@ -0,0 +1,28 @@
+# Script to rename the database name 
+
+The purpose of the script is to update the dashboard JSON template file with the database name of the user’s environment. 
+
+This script will generate the new upload JSON file with the database name of the user and will be ready to import to have the PGD dashboard.  
+
+Requires user to pass the database_name (which should be present in his/her environment) and optional input file and output file in JSON format as an argument in the script execution. The default extension is JSON for both input and output files. 
+
+# Requirements
+
+    Python 3.4 or higher
+
+# Run Comamnd
+
+    python3 db_name_change.py <database_name> -i <input_file> -o <output_file>
+    e.g.
+    python3 db_name_change.py edb -i utils/superset/pgd_monitoring_template.json  -o utils/superset/upload.json/
```

Comment:
```suggestion
    python3 db_name_change.py edb -i utils/superset/pgd_monitoring_template.json  -o utils/superset/upload.json
```

The output file is a file, not a directory, so, no trailing slashes at the end.

Reply:
Fixed. Thanks!